go/geth_adapter/adapter_test.go

```diff
@@ -44,6 +46,21 @@ func TestGethAdapter_RunContextAdapterImplementsRunContextInterface(t *testing.T
 	var _ tosca.RunContext = &runContextAdapter{}
 }
 
+func TestGethAdapter_NewGethInterpreterFactoryReturnsNonNilValues(t *testing.T) {
+	interpreter, err := lfvm.NewInterpreter(lfvm.Config{})
+	if err != nil {
```

Comment:
nit: since `require` is already imported in this file, it could be used instead of the conditions; but if this is the only thing, nevermind.